test/x/exercise_test.rb

```diff
@@ -0,0 +1,30 @@
+require_relative '../test_helper'
+require_relative '../x_helper'
+
+module X
+  class ExerciseTest < Minitest::Test
+    def test_test_files
+      f= './test/fixtures/xapi_v3_exercise_tests.json'
```

Comment:
The fixture shouldn't be named with `_tests` since all the fixtures are for the tests. Let's name them for the resource that they represent.

The equivalent fixtures for the `X::Track` tests are `xapi_v3_tracks.json` and `xapi_v3_track_fake.json`.